build/scripts/update-codegen.sh

```diff
@@ -7,7 +7,7 @@ set -o nounset
 SCRIPT_ROOT=$(git rev-parse --show-toplevel)
 
 # Grab code-generator version from go.sum.
-CODEGEN_VERSION=$(grep 'k8s.io/code-generator' go.sum | awk '{print $2}' | head -1)
+CODEGEN_VERSION=$(awk '/k8s.io\/code-generator/ {print $2}' go.sum | head -1 | cut -f 1 -d /)
 CODEGEN_PKG=$(echo `go env GOPATH`"/pkg/mod/k8s.io/code-generator@${CODEGEN_VERSION}")
```

Comment:
```suggestion
CODEGEN_PKG=$(go list -m -f '{{.Dir}}' 'k8s.io/code-generator')
```

Reply:
oh yeh that's much better, totally did not check where that version var was being used 😅